### DISCUSSION TITLE
Page title in browser not updating on navigation with SSR pages

### DISCUSSION THREAD

**Discussion by kevmarchant** [vercel/next.js#42601] [Help]
### Provide environment information

Operating System:
      Platform: darwin
      Arch: x64
      Version: Darwin Kernel Version 22.1.0: Sun Oct  9 20:14:54 PDT 2022; root:xnu-8792.41.9~2/RELEASE_X86_64
    Binaries:
      Node: 16.17.0
      npm: 8.15.0
      Yarn: 1.22.5
      pnpm: N/A
    Relevant packages:
      next: 13.0.2
      eslint-config-next: N/A
      react: 18.2.0
      react-dom: 18.2.0

### What browser are you using? (if relevant)

Chrome and Firefox at least

### How are you deploying your application? (if relevant)

Vercel, hooks from Github

### Describe the Bug

I am using simple <Head><title>Page title</title></Head> tags in each page on my site.  The whole site pre-renders on build and so all pages are SSR.  When you land on the first page, home for instance, the page title is correct.  Navigate to another page, and the title stays the same, although the underlying page source looks updated.  Reload the new page in the browser and the page title changes to the correct one.  It looks like the page title is not being updated on navigation.

Here is the website in question: https://naturallyhappydogs.com/

### Expected Behavior

Page title should change on page navigation

### Link to reproduction

https://naturallyhappydogs.com/

### To Reproduce

Simply navigate to https://naturallyhappydogs.com/ and click any of the navigation links

**Comment by timneutkens** [comment]
This site is on Next.js 9.5.5 🤔

**Comment by kevmarchant** [comment]
@timneutkens Should I upgrade Nextjs, might that solve the problem?

**Comment by timneutkens** [comment]
Maybe, wouldn't hurt to move it as this is quite an old version, latest is 13.0.2. Will convert this to discussions because it's not reproduced on a recent version.

**Reply by kevmarchant** [reply]
Thanks Tim, the issue is resolved with Next 13.x
